### PR TITLE
Add  DevOps team contact in the README

### PR DESCRIPTION
**Pull request by AbdelouahabMbarki**
Added communication channels to the README file #47
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,3 +95,15 @@ It provides a dashboard for Kubernetes and ks-devops.
 
 ## License
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2FLinuxSuRen%2Fks-devops.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2FLinuxSuRen%2Fks-devops?ref=badge_large)
+
+## Available communication channels:
+
+- [kubesphere Devops google group ](https://groups.google.com/g/kubesphere-sig-devops/)
+
```
Comment (LinuxSuRen) [inline README.md:102]: I prefer to keep these items be closer to each other. Please see the differences below.

image

image

I mean we don't need the extra space here.

File: README.md [hunk 2]
```diff
@@ -95,3 +95,15 @@ It provides a dashboard for Kubernetes and ks-devops.
 
 ## License
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2FLinuxSuRen%2Fks-devops.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2FLinuxSuRen%2Fks-devops?ref=badge_large)
+
+## Available communication channels:
+
+- [kubesphere Devops google group ](https://groups.google.com/g/kubesphere-sig-devops/)
```
Comment (LinuxSuRen) [inline README.md:101]: `kubesphere` -> `KubeSphere`

I prefer to remove the extra space here if you don't have a strong reason for that.

`[kubesphere Devops google group ]` -> `[kubesphere Devops google group]`

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
Thanks for helping with this. I found a few places that need to improve.

**Comment by AbdelouahabMbarki** [comment]
i added the review changes in the new commit

**Review by LinuxSuRen — APPROVED** [review]
/lgtm

**Pull request merged**